### ISSUE TITLE
harvestin not asking for advantage or disadvantage

### ISSUE THREAD

**Issue by tirzah2** [OhhLoz/Harvester#5]
Hi!
I think (but im not sure) that recent updates removed the possibility for the player to have adv or disadvantage when harvesting from a monster or beast. 
![image](https://github.com/OhhLoz/Harvester/assets/83312683/019848d7-03df-4512-932b-855519c9f5c6)

**Comment by OhhLoz** [comment]
Hi, 
You're right as there is no way for items to have attached skill checks, but there are ways of either:
Attacking, Healing, Ability Check, Ability Save or Custom Formula.

I used the custom formula and overwrote the button to display the skill check name and used the correct formula for the skills.
In the next update it might be possible to use a similar method as the previous updates after pressing the button and preventing the custom formula being executed, allowing the user to have the dice roll prompt. 

I tried having the popup using the custom formula however it's not possible so I will go back to the drawing board and try what I said above. If I can get that working then it's the best of both worlds.
